TaylorSource/CoreData/NSFRCDatasource.swift

```diff
@@ -0,0 +1,80 @@
+import UIKit
+import CoreData
+
+public struct NSFRCDatasource<
+    Factory
+    where
+    Factory: _FactoryType,
+    Factory.ViewType: NSFRCIndexedUpdateConsumer,
+    Factory.CellIndexType == NSFRCCellIndex,
+Factory.SupplementaryIndexType == NSFRCSupplementaryIndex>: DatasourceType {
+    
+    public typealias FactoryType = Factory
+    
+    public let identifier: String
+    public let factory: Factory
+    public var title: String? = .None
+    
+    public let updateHandler = NSFRCUpdateHandler()
+    public var selectionManager = IndexPathSelectionManager()
+    
+    public init(id: String, fetchedResultsController: NSFetchedResultsController, factory f: Factory) {
+        fetchedResultsController.delegate = self.updateHandler
+        self.fetchedResultsController = fetchedResultsController
+        identifier = id
+        factory = f
+    }
+    
+    public var numberOfSections: Int {
+        return self.fetchedResultsController.sections?.count ?? 0
```

Comment:
I've removed them. I agree it's probably best to match Swift's style, although I don't fully agree with that style. I'm not a big fan of it being easy to create a locally scoped var with the same name as an instance property. Also, visual noise notwithstanding, I find it slightly weird that for some contexts `self.` is required, but others it's omitted.